### PR TITLE
Salt Day 14 Normal Route PR

### PR DESCRIPTION
**Pull request by Parzivus**

### CODE REVIEW

**Comment by Parzivus** [comment]
Don't have time to fix this right now, I'll get it later tonight

File: script/Ciel/Day 14/14_01A_CIEL14A_1.txt [hunk 1]
```diff
@@ -64,7 +64,8 @@ I can hardly even believe it myself...
 [sha:e481fed7eba106aeb6ff46d70a3c9894857b9e5a]{
 -- Page 2, Offset 37614.
 -- 今の俺には、自分より大切な人がいる。
-But right now, I have someone I value more than my own life.
+// fitting in page
+But now, I have someone I value more than my own life.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01A_CIEL14A_1.txt:68]: Oops. Just fixed the allscr to remove the blank line on this page instead.

File: script/Ciel/Day 14/14_01_CIEL14_1.txt [hunk 2]
```diff
@@ -106,8 +106,9 @@ The human vessel cannot withstand such attrition,
 [sha:226b0316df7656509b12705329931abdaf56beb1]{
 -- Page 5, Offset 37023.
 -- 吸血種として不死の肉体を手にいれても、精神の鈍化は止められない。
-Even with the immortal body of a bloodsucker,
- one still wouldn't be able to prevent their mind from growing dull.
+// Trying to keep the next line on the page
+Even with the immortality of a bloodsucker,
+ one wouldn't be able to prevent their mind from growing dull.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01_CIEL14_1.txt:111]: Ok now this one I _had_ already fixed. 

2023-04-09-220521_2160x1286_scrot

File: script/Ciel/Day 14/14_01_CIEL14_1.txt [hunk 3]
```diff
@@ -390,7 +390,7 @@ It came from within.
 [sha:fd4697ea2adc972051226af55bf7c6d053790cfb]{
 -- Page 21, Offset 37074.
 -- 「う、ぼ…………………！」
-"%{i}Ugh%{/i}, %{i}cough%{/i}!"
+"%{i}Ugh%{/i}, %{i}cough%{/i}...!"
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01_CIEL14_1.txt:393]: Revert, no ... before !

File: script/Ciel/Day 14/14_01_CIEL14_1.txt [hunk 4]
```diff
@@ -414,7 +414,7 @@ Still bent over on the floor, I start vomiting.
 [sha:0243e1037a8ce71c09e59115a7cc74dddf469dd6]{
 -- Page 22, Offset 37078.
 -- 「あ―――、あ……！」
-"%{i}Gah%{/i}――%{i}ah%{/i}!"
+"%{i}Gah%{/i}――%{i}ah%{/i}...!"
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01_CIEL14_1.txt:417]: Revert, no ... before !

File: script/Ciel/Day 14/14_01_CIEL14_1.txt [hunk 5]
```diff
@@ -677,14 +677,14 @@ I know that.
 -- この頭痛が私の罠であろうとなかろうと、外に出れば俺には逃げ場はなくなるのだと。
 // 私 and 俺 used because it's half Roa
 // 逃げ場 is not literal
-Regardless of whether or not this headache is a trap set by%{i}me%{/i},
+Regardless of whether or not this headache is a trap set by% {i}me%{/i},
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01_CIEL14_1.txt:680]: space in the wrong place

**Pull request merged**